utils/utils.c

```diff
@@ -232,6 +235,30 @@ static bool can_remove_directory(const char *path)
 	return is_uftrace_directory(path) || is_empty_directory(path);
 }
 
+static bool create_default_opts(const char *dirname)
+{
+	char *opts_str = strv_join(&default_opts, " ");
+	char opts_filename[PATH_MAX];
+	FILE *fp;
+	bool ret = false;
+
+	snprintf(opts_filename, PATH_MAX, "%s/default.opts", dirname);
+	fp = fopen(opts_filename, "w");
+	if (fp == NULL) {
+		pr_dbg("Open failed: %s\n", opts_filename);
+		goto out;
+	}
+
+	fprintf(fp, "%s\n", opts_str ? opts_str : "");
```

Comment:
So you want to make this file even if no option is needed, right?  Then please call `fprintf()` only when the default_opts is set so that it can have 0 size if not used.

Reply:
done.